docs/proposals/PROPOSAL_A.md

```diff
@@ -0,0 +1,115 @@
+# Proposal A - FreeBSD Jails
+
+This proposal recommends changes to describe the requirements for mapping an OCI
+runtime container to a corresponding FreeBSD jail.
+
+## Modifications
+
+This suggests adding an object to the [FreeBSD-specific section](https://github.com/opencontainers/runtime-spec/blob/main/config.md#platform-specific-configuration) of the [container configuration](https://github.com/opencontainers/runtime-spec/blob/main/config.md) to describe the required parameters for the jail.
+
+## Jail Configuration
+
+Jail parameters to use for the container's jail.
+
+**`jail`** _(object, OPTIONAL)_ jail parameters for this container.
```

Comment:
Would there be anything directly in the `freebsd` object other than `jail`?

I'm imagining a structure based on this description like:

```json
{
  // standard platform-independent config
  "freebsd": {
    "jail": {
      // fields defined below
    }
  }
}
```

Reply:
In the latest iteration of the proposal, I added a "devices" member to the "freebsd" node